final_task/rss_reader/exceptions.py

```diff
@@ -0,0 +1,10 @@
+""" Module for non-standard exceptions. """
+
+
+class Error(Exception):
+    """ Class to raising exceptions. """
+    def __init__(self, message):
+        super().__init__(message)
+
```

Comment:
You don't really need this logic here. If you leave `pass` here it will do the same thing

Reply:
fixed